Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewversionstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View Versions Tab view class.
+     *
+     * @module ez-locationviewversionstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View View Versions tab class.
+     *
+     * @namespace eZ
+     * @class LocationViewVersionsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewVersionsTabView = Y.Base.create('locationViewVersionsTabView', Y.eZ.LocationViewTabView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireLoadVersions;
+            this._watchAttribute = 'versions';
+        },
+
+        render: function () {
+            var container = this.get('container'),
+                draft = [],
+                published = [],
+                archived = [],
+                hasDraft = false,
+                hasPublished = false,
+                hasArchived = false;
+
+            Y.Array.each(this.get('versions'), function (version) {
```

Comment:
splitting the versions by status is a tad too much logic for a render method. I think you should split the versions by status in `eZ.Content#loadVersions` so that its callback directly receives an object where the versions are already sorted out.

Here you would just have to iterate over the 3 arrays to call `toJSON` and maybe to compute the `has*` flags.